From e337999155209ab2a209ec7868921e8f2f5d3d8a Mon Sep 17 00:00:00 2001
From: Chao Zeng <chao.zeng@siemens.com>
Date: Wed, 6 Jan 2021 15:51:18 +0800
Subject: [PATCH 02/11] Workaround to correct DP clock to 154MHZ.

BUG: as the default DPI source should be DPI_1_IN_CLK
     but seems the DSS input clk swap the DPI0 and DPI1
     so change the DPI clock to DPI0 to provide the clock
     to DPI

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
---
 arch/arm64/boot/dts/ti/k3-am65-iot2050.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am65-iot2050.dtsi b/arch/arm64/boot/dts/ti/k3-am65-iot2050.dtsi
index 5ffcbd9641b7..0dba4e95ca90 100644
--- a/arch/arm64/boot/dts/ti/k3-am65-iot2050.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am65-iot2050.dtsi
@@ -729,9 +729,9 @@
 
 	pinctrl-names = "default";
 	pinctrl-0 = <&dss_vout1_pins_default>;
-
-	assigned-clocks = <&k3_clks 67 2>;
-	assigned-clock-parents = <&k3_clks 67 5>;
+	// Workaround to slove the DP clock,change the Clock to DPI_0
+	assigned-clocks = <&k3_clks 67 0>;
+	// assigned-clock-parents = <&k3_clks 67 5>;
 };
 
 &dss_ports {
-- 
2.31.1

